### ISSUE TITLE
Docker cannot start anymore with version 2.0.0.0-win78

### ISSUE THREAD

**Issue by Thierry61** [docker/for-win#2941]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: FF57B48F-BEE5-4A33-8329-3D8D3C1DB348/20181121104503

### Expected behavior

Starting docker should work.

### Actual behavior

After installing version 2.0.0.0-win78 docker cannot start anymore:

```
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.39/containers/json: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

   at Docker.Core.Pipe.NamedPipeClient.Send(String action, Object[] parameters) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeClient.cs:line 36
   at Docker.Actions.DoStart(SynchronizationContext syncCtx, Boolean showWelcomeWindow, Boolean executeAfterStartCleanup) in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 92
   at Docker.Actions.<>c__DisplayClass19_0.<Start>b__0() in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 74
   at Docker.WPF.TaskQueue.<>c__DisplayClass19_0.<.ctor>b__1() in C:\workspaces\stable-18.09.x\src\github.com\docker\pinata\win\src\Docker.WPF\TaskQueue.cs:line 59
```
Then I put back version 18.06.1-ce-win73 and it works again flawlessly, so the problem is a regression brought by new version.

### Information
  - The problem is reproducible
  - The problem is new.
  - The problem appeared with update 2.0.0.0-win78
  - Windows Version: Windows 10 Pro version 1709 (OS Build 16299.726)
  - Docker for Windows Version: 2.0.0.0-win78
  - Workaround: Install Docker for Windows version 18.06.1-ce-win73

### Steps to reproduce the behavior
  1. Start docker
  2. (no further steps)

**Comment by homimo** [comment]
I had same problem. 
If you use storage-driver aufs.
Storage-driver aufs is deprecated in 2.0.0.0-win78. Changing its value to overlay2 solved my issue.

**Comment by Trophime** [comment]
@homimo could you explain more precisely what has to be done to get docker working?
Thanks

**Comment by homimo** [comment]
@Trophime in my case I go to docker settings -> Daemon
here switch to advanced and change line (I already use it with "storage-driver": "aufs"  value) to
  "storage-driver": "overlay2"

then restart docker.

**Comment by Thierry61** [comment]
Not my case because I am using overlay2.

**Comment by choux130** [comment]
@Thierry61 Thank you! I downgraded to Version 18.06.1-ce-win73 and things work again!! 
Here are the steps I did, 
1. Uninstall Docker (https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon#uninstall-docker)
2. Download the old version, Docker Community Edition 18.06.1-ce-win73 2018-08-29 (https://docs.docker.com/docker-for-windows/release-notes/#stable-releases-of-2018)
3. Install and it works again!
